### DISCUSSION TITLE
Validating a field without including it into values

### DISCUSSION THREAD

**Discussion by BasixKOR** [react-hook-form/react-hook-form#6406] [Q&A] [answered]
Hi, I'm trying to make a simple 'confirm password' field and validate it using this. However it requires me to register the field to the form, which will be included in submitted data.

I want to validate my value but exclude it from the form values. Is it possible to do so? `disabled` seems like the closest option, but it actually disables the field as well.

**Comment by bluebill1049** [comment, marked as answer]
just don't register that confirmed password input, only make it trigger validation.

**Reply by BasixKOR** [reply]
But how do I validate my `confirm password' field in that case?

**Reply by bluebill1049** [reply]
```tsx
<input {...register('a', { validate: () => { // test if a === b } })} />

<input onChange={() => trigger('a')} />
```
